README.md

```diff
@@ -104,5 +105,12 @@ To trigger [the release workflow](https://github.com/veezor/maestro/blob/main/.g
 1. Create a pull request for your changes or bug fixes. This will deploy a pre-release image that can be tested with the proposed changes.
 2. Create a new tag and a new release, following [semantic versioning parameters](https://semver.org/).
 
+### How to register a deploy with a webhook
+Maestro permit register a deploy with a webhook, but it's necessary has a environment variable based on a default (DEPLOY_WEBHOOK_URL), that need's contains in the URL:
```

Comment:
Copy changes suggestion:

Maestro allows you to register a deploy with a custom webhook using a standard `GET` request. To trigger that, you need to define `DEPLOY_WEBHOOK_URL` environment variable and its value may contain the following placeholders within the URL:

- `{{CLUSTER}}` for the cluster's name
- `{{SERVICE}}` for the service's name
- `{{REPOSITORY}}` for the repository's slug